CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v4.1.1](https://github.com/voxpupuli/puppet-snmp/tree/v4.1.1) (2019-02-07)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-snmp/compare/v4.1.0...v4.1.1)
+
+**Breaking changes:**
+
+- modulesync 2.5.1 and drop Puppet 4 [\#177](https://github.com/voxpupuli/puppet-snmp/pull/177) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
Dropping support for puppet 4 I think is being treated as breaking.
@bastelfreak?

Reply:
correct. this is listed as breaking change so it should be a 5.0.0 release.